ratpack-kafka-consumer/src/main/java/smartthings/ratpack/kafka/KafkaConsumerModule.java

```diff
@@ -0,0 +1,42 @@
+package smartthings.ratpack.kafka;
+
+import com.google.inject.Scopes;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import ratpack.guice.ConfigurableModule;
+
+import java.util.Properties;
+import java.util.Set;
+
+public class KafkaConsumerModule extends ConfigurableModule<KafkaConsumerModule.Config> {
+
+	protected void configure() {
+		bind(KafkaConsumerService.class).in(Scopes.SINGLETON);
+	}
+
+	public static class Config {
```

Comment:
I reserve my right to change my mind with the wind, but at the moment I think it makes more sense to combine the two sub-projects.  It better fits in w/ functionality already in kafka-client, and as a user of the lib I'd rather just configure 1 Guice module.  The amount of code is quite small, and even if you're only using one the code in the unused portion is pretty much inert.